### ISSUE TITLE
Expose peer certificates on MqttEndpoint

### ISSUE THREAD

**Issue by sauthieg** [vert-x3/vertx-mqtt#19]
Just like HTTP connection, when SSL client authentication is enabled, that's useful to grab the peer (client) certificates.

That information is available on `ConnectionBase`, but not exposed through the `MqttEndpoint` interface.

**Comment by vietj** [comment]
I'm assuming this was released in 3.4.2 ?

**Comment by sauthieg** [comment]
No, there is still an ongoing PR (#42) hold by @dachaac about SSL API exposure: so not in 3.4.2, not even in 3.5.0 ...

Originally, I was only interested in an API to get the client certificates, but #42 is a little bit richer than that.

**Comment by sophokles73** [comment]
This is a much anticipated feature from our side :-)
